README.md

```diff
@@ -87,9 +87,6 @@ orphaned due to crashes.
   independently at runtime locally or across _all_ running nodes (even in
   environments like Heroku, without distributed Erlang).
 
-- **Resilient Queues** — Failing queries won't crash the entire supervision tree,
-  instead they trip a circuit breaker and will be retried again in the future.
-
```

Comment:
This is still true, but it is done with a fine-grained retry/backoff mechanism rather than circuit breakers. A few examples:

https://github.com/sorentwo/oban/blob/main/lib/oban/queue/executor.ex#L89
https://github.com/sorentwo/oban/blob/main/lib/oban/queue/producer.ex#L108

Pro's `SmartEngine` wraps all of the queue operations in a similar retry/backoff.

Would you mind making the paragraph more accurate, rather than removing it?